k8s/helmfile/env/production/sql.values.yaml.gotmpl

```diff
@@ -1,3 +1,18 @@
+image:
+  registry: ghcr.io
+  repository: wbstack/mariadb
+  tag: 10.5.15-debian-10-r52
+
+# see https://github.com/wbstack/bitnami-legacy/blob/mariadb/10.5.0/bitnami/mariadb/values.yaml#L928
+volumePermissions:
+  # Depends on an image which may no longer exist
+  enabled: false
+
+# see https://github.com/wbstack/bitnami-legacy/blob/mariadb/10.5.0/bitnami/mariadb/values.yaml#L961
+metrics:
+  # Depends on an image which may no longer exist
+  enabled: false
```

Comment:
How come these need to be specified? The defaults that you linked to for `enabled` are `false` for both.

Reply:
Good to know, thanks